README.md

```diff
@@ -0,0 +1,37 @@
+# CNOE Stacks
+
+This repository contains building blocks and examples to help you build your own Internal Developer Platform.
+
+## Getting Started
+
+### Install idpbuilder
+
+To get started, you need to install [`idpbuilder`](https://github.com/cnoe-io/idpbuilder).
+
+The following command can be used as a convenience for installing `idpbuilder`, (be sure to check the script first if you are concerned):
```

Comment:
I think you can remove everything here down until `Using this repository`. Otherwise, every time the instructions in `idpbuilder` are updated, you'll want to copy/paste them here, no?

Reply:
Good point!